src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIBookingService.kt

```diff
@@ -129,6 +132,7 @@ data class AppointmentRescheduleRequestDTO(
   val updatedAppointmentStart: OffsetDateTime?,
   val updatedAppointmentEnd: OffsetDateTime?,
   val initiatedByServiceProvider: Boolean,
+  val officeLocationCode: String?,
```

Comment:
Does this still need to be nullable?

Reply:
Yes, the reason I made it nullable here was to reflex the usage of the DTO on `CommunityApi` which allows nullable.

I agree that it should be non-nullable as that was a backwards compatible change you did.

(I think also updateAppointmentStart and End should be non-nullable too)